packages/slice-machine/components/ChangesEmptyPage/AuthErrorPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Button, Flex, Text } from "theme-ui";
+
+export const AuthErrorPage = () => {
+  return (
+    <Flex sx={{ justifyContent: "center", alignItems: "center", flex: 1 }}>
+      <Flex
+        sx={{
+          flexDirection: "column",
+          alignItems: "center",
+          transform: "translateY(-50%)",
```

Comment:
interesting! what does this do?

Reply:
moves the content of the page slightly up (by half of its height so that it does not reply on sizes or pixels). Was because the figma design does not have the page content centered, and this is what me and Louis though looked best when pairing on the design :)